azurelinuxagent/common/osutil/default.py

```diff
@@ -1411,6 +1412,30 @@ def get_total_cpu_ticks_since_boot():
                     break
         return system_cpu
 
+    @staticmethod
+    def get_used_and_available_system_memory():
+        """
+        Get the contents of free -b in bytes.
+        # free -b
+        #              total        used        free      shared  buff/cache   available
+        # Mem:     8340144128   619352064  5236809728     1499136  2483982336  7426314240
+        # Swap:             0           0           0
+
+        :return: used and available memory in megabytes
+        """
+        used_mem = available_mem = 0
+        free_cmd = ["free", "-b"]
+        try:
+            memory = shellutil.run_command(free_cmd)
+            for line in memory.split("\n"):
+                if ALL_MEMS_REGEX.match(line):
+                    mems = line.split()
+                    used_mem = int(mems[2])
+                    available_mem = int(mems[6])  # see "man free" for a description of these fields
```

Comment:
available would be 7

Reply:
yes, that's why I used index `'6'`to get that.

Note: List start index is '`0`'

Reply:
0 is the comment, total is 2, available 7

Reply:
or you added the comment and total should be 1?

Reply:
oh, used, not total --- never mind, you are ok